### PR TITLE
Add Wget dependency

### LINKED ISSUE #227: Wget should be a zero dependency
`zero check` should include wget as a dependency.

### PR DESCRIPTION
**Pull request by dtoki**
Adds Wget as a zero dependency

closes #227

### CODE REVIEW

File: cmd/check.go [hunk 1]
```diff
@@ -143,6 +143,14 @@ var checkCmd = &cobra.Command{
 				minVersion: "2.17.1",
 				docsURL:    "https://git-scm.com/book/en/v2/Getting-Started-Installing-Git",
 			},
+			{
+				name:       "Wget\t\t",
+				command:    "wget",
+				args:       []string{"--version"},
+				regexStr:   `^GNU Wget (0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)`,
+				minVersion: "1.20.3",
```
Comment (davidcheung) [inline cmd/check.go:151]: looks like 1.20.3 is the latest one, people may not update their wget often
might be safer to go with some older versions as I doubt there's many breaking changes on wget

do you think it'll be good to allow some older versions too?

https://ftp.gnu.org/gnu/wget/?C=M;O=D

File: cmd/check.go [hunk 2]
```diff
@@ -143,6 +143,14 @@ var checkCmd = &cobra.Command{
 				minVersion: "2.17.1",
 				docsURL:    "https://git-scm.com/book/en/v2/Getting-Started-Installing-Git",
 			},
+			{
```
Comment (davidcheung) [inline cmd/check.go:146]: Can you also bump the terraform to 0.13 please, the EKS stack's terraforms are now using some 0.13 features

**Comment by bmonkman** [comment]
Can you also please add it as a dependency for brew in `.goreleaser.yml`?

File: cmd/check.go (hunk 1 above)
Comment (dtoki) [reply cmd/check.go:151]: We're all probably using and testing on`1.20.3` but I'm comfortable supporting a minor version back `1.19.0` released in 2017

Comment (davidcheung) [reply cmd/check.go]: yeah 2012 definitely seems far back enough, I think as long as it covers most people's new computer cycle then its good 👍

**Pull request merged**